tests/regression/56-asm/60-goto-issue.c

```diff
@@ -0,0 +1,45 @@
+// see https://github.com/goblint/cil/issues/81
```

Comment:
Is this test currently only about parsing and not crashing Goblint? Maybe add an explicit comment about that, and the current limitations.